python/lsst/meas/algorithms/sourceSelector.py

```diff
@@ -31,6 +31,8 @@
 import abc
 import numpy as np
 import astropy.units as u
+from pandas import DataFrame
+from astropy.table import Table
```

Comment:
Please make this `import astropy.table` and use `astropy.table.Table` below: the `Table` name is too generic to be a top level import without a namespace. I'd almost say the same for `DataFrame`, but I think that only exists in pandas.